### DISCUSSION TITLE
MSnoise problem - stack

### DISCUSSION THREAD

**Discussion by malmassri** [ROBelgium/MSNoise#313] [General]
Hello,

Since I am new to MSnoise, I have some challenges in the process. 

I computed the cross-correlation then I tried to stack but it gave me an error saying: " Error: No such command 'stack'. "

Any suggestions or comments are greatly appreciated.

**Comment by ThomasLecocq** [comment]
Are you using the new/development version? If yes, the command is msnoise cc stack (see http://msnoise.org/doc/master)

**Reply by malmassri** [reply]
Yes. But I restacked again and received this:
UserWarning: msnoise could not determine its version number. Make sure it is properly installed. This for example happens when installing from a zip archive of the msnoise repository which is not a supported way of installing msnoise.

**Reply by ThomasLecocq** [reply]
This is just a warning, it doesn't prevent msnoise from working ok.

**Reply by malmassri** [reply]
Yes but I do not get the outcome of the stacking

**Reply by malmassri** [reply]
In terminal:  msnoise stack --help

Error: No such command 'stack'.

**Reply by ThomasLecocq** [reply]
I just wrote "msnoise cc stack" above, please read and check the doc related to the Dev version

**Reply by malmassri** [reply]
Thank you for your reply. 
I have used that command but still have an issue. I will go through the workflow again and try to solve the issue.

Cheers.

**Reply by malmassri** [reply]
Hi Thomas,

I attempted to graph the MWCS, but it was unsuccessful. I suspect the issue is with the format of the outcome file [ .nc] while the code is reading the [ .txt] file. 

Also, I tried computing DVV using "msnoise cc dvv compute_dvv" before the plot "dv/v", but it did not function either. 
>> The error message::: AttributeError: 'Series' object has no attribute 'stack'. Did you mean: 'unstack'?

Your thoughts and suggestions are greatly appreciated in this matter.

**Reply by ThomasLecocq** [reply]
yes, the plots in the dev version were not (maybe still not) working with the new file structure...
